cmd/renterd/main.go

```diff
@@ -96,20 +98,30 @@ func main() {
 		node.AutopilotConfig
 	}
 
+	// NOTE: the flag package does not offer a nice way to parse currencies with
+	// a fallback to a default value, flag.TextVar would work but that was added
+	// in Go1.19
+	var flagg utils.CurrencyFlags
```

Comment:
what about implementing `flag.Value`?

Reply:
can't easily default :(

Reply:
That was my first version but there are a couple of shortcoming when you use `flag.Value`, mostly related to the default value. You don't know whether the user passed the flag if you use `flag.Value`, and you don't have a clean way to default the value. I also did not want to wrap `types.Currency`.

Are you against the `CurrencyFlags`? It's temporary and I personally think it's not too bad? At least it's super consistent with the other flags:
```go
flag.Uint64Var(&busCfg.TotalShards, "bus.totalShards", 30, "total amount of ...")
flagg.CurrencyVar(&busCfg.MaxRPCPrice, "bus.maxRPCPrice", types.SiacoinPrecision, "max allowed ...")
```

Reply:
> You don't know whether the user passed the flag

You don't know this with standard flags anyway, but it rarely matters.

Also, `flagg` is the name of [a package](https://lukechampine.com/flagg) that we might use :)

I think wrapping is the way to go. I've pushed a commit implementing this approach.

Reply:
Oh that is way better. It's shameful stuff but I didn't consider setting default initially so I thought I'd needed a check to see if the flag was passed.